pkgs/development/python-modules/webargs/default.nix

```diff
@@ -0,0 +1,42 @@
+{ buildPythonPackage, fetchPypi, lib, isPy27, marshmallow, pytest
+, pytest-aiohttp, webtest, webtest-aiohttp, flask, django, bottle, tornado
+, pyramid, falcon, aiohttp }:
+
+buildPythonPackage rec {
+  pname = "webargs";
+  version = "6.1.0";
+  disabled = isPy27;
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0gxvd1k5czch2l3jpvgbb53wbzl2drld25rs45jcfkrwbjrpzd7b";
+  };
+
+  checkPhase = ''
+    rm tests/test_webapp2parser.py  # webapp2 doesn't support python 3
+    pytest
+  '';
```

Comment:
```suggestion
  preCheck = ''
    rm tests/test_webapp2parser.py  # webapp2 doesn't support python 3
  '';
```

Reply:
This file was already removed in webargs 8.0.0, so there is no need to remove it from the Nix expressoin.